scratch/test-rank-4.ts

```
@@ -0,0 +1,101 @@
+import { assertBotGate } from '../src/lib/security/bot-gate';
+import { 
+  withGovernanceEnvelope, 
+  assertHumanFinalized, 
+  isHighStakes, 
+  AI_DISCLOSURE_NOTICE,
+  HIGH_STAKES_DOMAINS 
```

ESLint

[error] 'HIGH_STAKES_DOMAINS' is defined but never used.

(unused-imports/no-unused-imports)